static/js/main.js

```diff
@@ -5,6 +5,8 @@ window.shark = window.shark || {};
 window.shark.info = window.shark.info || {};
 window.shark.screenshots = window.shark.screenshots || {};
 
+var screenshot_counter = 0;
```

Comment:
Stash this in `window.shark.info`.

That aside, you should almost never be using `var`.
`var` will hoist declarations to the top of the function context, which is typically not what you want.
Favor `let` or `const`.